src/Documentation/glossary.js

```diff
@@ -5,15 +5,15 @@ export default {
     'relevant DVC concepts and terminologies which are frequently used.',
   contents: [
     {
-      name: 'Workspace directory',
+      name: 'DVC Workspace',
```

Comment:
let's keep the previous one. Actually, we should change the description below. I think we should be emphasizing DVC here. For example, I can have workspace (Git repo, or even without Git) with my code and data without any DVC. So, workspace is broader, more generic term. It contains code, data, models, DVC-files if DVC is enabled.

Reply:
Isn't calling it "workspace directory" redundant? The workspace IS a directory. We never call it that in the docs. I vote for just "Workspace" but lmk if you prefer "Workspace Directory" and I'll restore it.

Updated description, better? Not sure I'm understanding your comment 100%.

Reply:
Oh... So you mean the "workspace" is a general concept that may or may not include DVC? That would be a new way of understanding the term for me! I thought we only call it "workspace" in the context of an existing DVC project (a directory that has been `dvc init`ialized).

Reply:
Ping @shcheklein

Reply:
hey! yep, `directory` is redundant for sure.

Workspace is a more general term than DVC project.

For example, in get started it's pretty reasonable to say:

`Without DVC you have to manually put data in your workspace into .gitignore.`

Reply:
I see. Updated in 686561b but I may need to review also all the existing uses of the term "workspace" to make sure its meaning is consistent...

p.s. Also made some updates to the get-started including your note above in fc28eb0.